### ISSUE TITLE
Reopen issue #129

### ISSUE THREAD

**Issue by stevvik** [The-OpenROAD-Project/OpenSTA#134]
Hello,
I want to pay your attention to my comments to closed issue #129:
https://github.com/The-OpenROAD-Project/OpenSTA/issues/129#issuecomment-1335453272

This issue was not fixed correctly enough from my point of view and I explained the problem in comment.
But still not any reply.
Looks like you missed this comment. Did you?
Sorry if I'm wrong.

**Comment by jjcherry56** [comment]
No, I did not miss it. I pointed out that you need to define the clocks with the SDC commabnd create_clock rather than expect the tool to infer them.
